restart.sh

```diff
@@ -0,0 +1,16 @@
+
+#!/bin/bash
+set -euo pipefail
+
+export KUBE_NAMESPACE=${ENVIRONMENT}
+
+if [[ ${KUBE_NAMESPACE} == *prod ]]
+then
```

Comment:
This is more conventional (in our codebase, at least):
```suggestion
if [[ ${KUBE_NAMESPACE} == *prod ]] ; then
```

Reply:
I copied the format from the deploy - I have changed this as I tend to agree.